File: examples/python/src/list_sims.py
import os
from unlimited_messaging import UnlimitedMessagingApi

client = UnlimitedMessagingApi(
    base_url="https://api.unlimitedmessaging.app",
    token=os.environ["API_TOKEN"],
)

# Get all linked SIMs
sims = client.sim.get_linked_sims()

for sim in sims:
    print(f"SIM {sim.id} — {sim.phone or 'no phone'}")

File: examples/typescript/list-sims.ts
import { UnlimitedMessagingApiClient } from "@unlimited-messaging/sdk";

const client = new UnlimitedMessagingApiClient({
  environment: "https://api.unlimitedmessaging.app",
  token: process.env.API_TOKEN!,
});

// Get all linked SIMs
const sims = await client.sim.simControllerGetLinkedSims();

for (const sim of sims) {
  console.log(`SIM ${sim.id} — ${sim.phone ?? "no phone"}`);
}
